app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/claims/check_your_answers.scala.html

```diff
@@ -60,8 +59,10 @@
         @pageHeading(messages(s"$key.claimant-details.h2"), "govuk-heading-m govuk-!-margin-top-9", "h2")
         @summary(SummaryList(cyaHelper.makeClaimantDetailsSummary(completeClaim)))
 
-        @pageHeading(messages(s"$key.reason-basis.h2"), "govuk-heading-m govuk-!-margin-top-9", "h2")
-        @summary(SummaryList(cyaHelper.makeBasisAndOrReasonForClaim(SelectBasisForClaimController.key, completeClaim)))
+    @completeClaim.maybeBasisOfClaimAnswer.map { basisOfClaim =>
+        @pageHeading(messages(s"$key.basis.h2"), "govuk-heading-m govuk-!-margin-top-9", "h2")
+        @summary(basisOfClaim.review(s"$key.basis"))
+    }
```

Comment:
Can you please align 
```
    @completeClaim.maybeBasisOfClaimAnswer.map { basisOfClaim =>
        @pageHeading(messages(s"$key.basis.h2"), "govuk-heading-m govuk-!-margin-top-9", "h2")
        @summary(basisOfClaim.review(s"$key.basis"))
    }
```
to be consistent with the rest of code. I mean use exact number of the spaces from the left.